lookout/style/format/rules.py

```diff
@@ -589,3 +590,21 @@ def _sanitize_params(params: Dict[str, Any]) -> Dict[str, Any]:
                 v = list(v)
             sanitized[k] = v
         return sanitized
+
+    def _apply_config(self, config: Optional[Dict[str, Any]] = None) -> Dict[str, Any]:
```

Comment:
if we use the `__setitem__` method to get the config to modelforge, we can forgo this method and directly **config[key] they relevant key to initialize TrainableRules

Reply:
like this: https://github.com/src-d/style-analyzer/pull/152#discussion_r220448150